kmip/tests/unit/core/attributes/test_digest.py

```diff
@@ -316,9 +318,17 @@ def test_not_equal_on_not_equal(self):
             digest_value=self.digest_value_b,
             key_format_type=self.key_format_type_b)
         b = Digest()
+        c = Digest(
+            hashing_algorithm=self.hashing_algorithm_d,
+            digest_value=self.digest_value_b,
+            key_format_type=self.key_format_type_c)
+        d = Digest(
+            key_format_type=self.key_format_type_c)
 
         self.assertTrue(a != b)
         self.assertTrue(b != a)
+        self.assertTrue(b != c)
+        self.assertTrue(b != d)
```

Comment:
What's the purpose of these assertions?

Why is `b` compared to `c` and `d` but `a` is not compared to all the others?